src/components/CommunityEvents/index.tsx

```diff
@@ -96,13 +100,13 @@ const CommunityEvents = () => {
       <Center w={{ base: "100%", lg: "40%" }}>
         <Box pe={8} ps={{ base: 8, lg: 0 }}>
           <OldHeading>
-            <Translation id="community-events-content-heading" />
+            <Translation id="page-index:community-events-content-heading" />
           </OldHeading>
           <Text>
-            <Translation id="community-events-content-1" />
+            <Translation id="page-index:community-events-content-1" />
```

Comment:
I don't think this can happen right now. The t() way of doing things doesn't render html elements in the strings properly, and the Translation component does.